registry/app.py

```diff
@@ -38,6 +38,19 @@ def page_not_found(e):
     def format_time(date, format="%d.%m.%Y %H:%M:%S"):
         return date.strftime(format)
 
+    @app.template_filter("translate")
+    def translate(input, rodne_cislo):
+        word_map = {
+            "pracovník": "pracovnice",
+            "p.": "pí.",
```

Comment:
```suggestion
            "p.": "pí",
```
Dle [Internetové jazykové příručky](https://prirucka.ujc.cas.cz/?id=p%C3%AD_1) by měla být pro _paní_ přípustná i zkratka `p.`, v případě, že použijeme `pí`, tak se to píše bez tečky na konci.